mautic/mautic
# Add contact ID to plugin mapping

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | 
| New feature? | yes
| Automated tests included? |
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | none
| Deprecations? | none

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
This PR add the contact ID in plugin mapping. That can help to send mautic contact ID into a CRM for instance.
**⚠️  This PR comes in favor of https://github.com/mautic/mautic/pull/4316 that added the ID everywhere in the app and was dangerous (in campaign action "update contact fields" for instance).**

[//]: # ( As applicable: )
#### Steps to test this PR:
1. Apply the PR
2. Configure a plugin
3. Go to contact field mapping
4. Notice that the mautic contact ID is available
